src/senaite/core/z3cform/widgets/address.py

```diff
@@ -297,8 +301,7 @@ def to_dict(subdivision):
                 "code": subdivision.code,
                 "type": self.context.translate(_(subdivision.type)),
             }
-        items = map(to_dict, items)
-        return json.dumps(items)
+        return map(to_dict, items)
```

Comment:
better wrap in a `list`, because Python 3 returns an generator for `map` and I'm not sure if it is JSON serializable

Reply:
noted!